docs/developer/development.rst

```diff
@@ -36,7 +42,7 @@ Fetch the data from GitHub (download it to your local directory):
 
 .. code-block:: bash
 
-  git fetch
+  git fetch # [--all] [--prune]
```

Comment:
I suggest to embed these changes in some explanation, since inexperienced users would copy/retype the whole line, which would introduce some overhead.
```suggestion
  git fetch
```